master/docs/manual/configuration/www.rst

```diff
@@ -131,6 +131,8 @@ This server is configured with the ``www`` configuration key, which specifies a
         * quotes in attributes are not quoted. https://github.com/syrusakbary/pyjade/issues/132
```

Comment:
we should probably make a quick test and see if this is still the case.

Reply:
OK, the new code is identical aside from some minor reformatting. So, no it has not been fixed.

The new code is at lines 145 to 155 of nodes.py